Source/NSTextView.m

```diff
@@ -6078,6 +6081,88 @@ - (void) performFindPanelAction: (id)sender
 		withTextView: self];
 }
 
+// NSTextFinder methods implementation...
+// isSelectable, isEditable, string, selectedRanges, setSelectedRanges, replaceCharactersInRange:withString:
+// implemented by NSTextView already...
+
+- (BOOL) allowsMultipleSelection
+{
+  return NO;
+}
+
+- (NSString *) stringAtIndex: (NSUInteger)characterIndex
```

Comment:
For NSTextView it would actually be better not to implement the next two methods. In that case the NSTextFinder would just use the "string" method. But if you really want to implement those, this method here should return the full string of the NSTextView.

Reply:
Done.